src/components/Footer/Footer.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { FOOTER_DATA_1, FOOTER_DATA_2 } from './FooterData';
+import '/images/logo.svg';
+
+const Footer = props => (
+  <FooterContainer>
+    <FooterWrapper>
+      <Link to="/">S T A Y F O L I O</Link>
+
+      <FooterInfo>
+        <Ul1>
+          {FOOTER_DATA_1.map(data => {
+            return <List key={data.id}>{data.value}</List>;
+          })}
```

Comment:
```suggestion
          {FOOTER_DATA_1.map(data => <List key={data.id}>{data.value}</List>)}
```
arrow function 문법중에서 생략 가능한 부분 찾으면 위와 같이 간결하게 표현할수도 있습니다!!

그리고 data 등 너무 범용적인 이름을 변수명으로 사용하는 것은 지양해주세요 가장 기본적이지만 기본이기에 그만큼 중요한 부분입니다.

Reply:
오 감사합니다! 만들기에 급급해서 제일 중요한 네이밍 컨벤션을 중요하게 다루지 않았네요 다시 명심하겠습니다!